QuickFIXn/SessionState.cs

```diff
@@ -290,6 +290,11 @@ public void Get(int begSeqNo, int endSeqNo, List<string> messages)
             }
         }
 
+        public void GetEnumerable(int begSeqNo, int endSeqNo)
```

Comment:
This PR as-is doesn't build.
```suggestion
        public IEnumerable<string> GetEnumerable(int begSeqNo, int endSeqNo)
```